.github/workflows/dipatch-update.yml

```diff
@@ -0,0 +1,33 @@
+name: Dispatch stations update
+on:
+  push:
+    branches: [master]
+    paths:
+      - stations.csv
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    environment: main
+    steps:
+      - name: Generate GitHub App token
+        id: generate-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ vars.APP_STATION_SYNC_ID }}
+          private-key: ${{ secrets.APP_STATION_SYNC_PRIVATE_KEY }}
+          owner: trainline-private
+          repositories: |
+            stations
+
+      - name: Stations updated dispatch
+        uses: peter-evans/repository-dispatch@v3
```

Comment:
🤔 I'm surprised github is not providing github-owned action for that

Reply:
It was suggested in multiple places and indeed GitHub does not offer the own action for this.